source/portable/mbedtls/core_pkcs11_mbedtls.c

```diff
@@ -4986,6 +4988,8 @@ CK_DECLARE_FUNCTION( CK_RV, C_Verify )( CK_SESSION_HANDLE hSession,
                         }
                     }
                 }
+
+                prvHMACCleanUp( pxSessionObj );
```

Comment:
Could we add a cmock stub for alloc and free functions and catch this in unit tests?

Reply:
That is a good idea. I'll see separately if that is possible.